plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java

```diff
@@ -707,4 +706,22 @@ private Object readResolve() {
     private static final Logger LOGGER = Logger.getLogger(DescribableModel.class.getName());
 
     private static final long serialVersionUID = 1L;
+
+    /**
+     * Given the primitive type, returns the VM default value for that type in a boxed form.
```

Comment:
Please file a PR for Jenkins core that fixes the bug in `ReflectionUtils`, and add a TODO comment here linking to that so in the future we could switch back to the standard version.